index.html

```diff
@@ -2,18 +2,16 @@
 <html>
 
     <head>
-        <title>Login - Spotify</title>
         <link rel="stylesheet" type="text/css" href="style.css" />
+        <title>Login - Spotify</title>
     </head>
 
     <body> 
 
         <div class="topnav">
-            <div class="navlogo">
```

Comment:
There was no need for this div, I deleted it and nothing changed, remember no unnecessary extra code 😁